packages/core/src/odata/common/request/odata-request.ts

```diff
@@ -194,19 +194,18 @@ export class ODataRequest<RequestConfigT extends ODataRequestConfig> {
    * @returns Key-value pairs where the key is the name of a header property and the value is the respective value
    */
   defaultHeaders(): Record<string, any> {
-    const defaultHeaders = replaceDuplicateKeys(
-      filterNullishValues({
-        accept: 'application/json',
-        'content-type': this.config.contentType
+    const customDefaultHeaders = Object.keys(this.config.defaultHeaders).reduce(
```

Comment:
It took me a moment to realize what happens here. What about adding a new util method `getHeaders` where you can put in a list of keys and get the headers related ignoring case of the key in the search.

Reply:
I can do that.